joplin/api/schema.py

```diff
@@ -639,6 +668,9 @@ def resolve_title(self, resolve_info, *args, **kwargs):
     def resolve_slug(self, resolve_info, *args, **kwargs):
         return get_page_from_content_type(self).slug
 
+    def resolve_page_type(self, info):
+        return self.page.content_type.name
+
```

Comment:
When getting all the `TopicPageTopPages` from a specific Topic Collection, we need to also add the page type to the api so that it can do specific things on Janis based on what page it is. Like guide pages with guide images.